x-pack/plugins/observability_solution/logs_shared/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -222,6 +222,7 @@ export class ScrollableLogTextStreamView extends React.PureComponent<
                                 onExtendRange={(newDateExpression) =>
                                   updateDateRange({ startDateExpression: newDateExpression })
                                 }
+                                aria-owns="streamEntry logTextStreamEntry"
```

Comment:
I took a look at the originating issue again. I think what Bhavya was reporting was the Logs `div[role="row"]` "table rows" need a parent `role="table"`. I quickly adjusted line 160 and added that role, then ran axe-core again. The error went away after I added the role and added `aria-hidden="true"` to both HR tags.

```diff
! scrollable_log_text_stream_view.tsx#160

+ <ScrollableLogTextStreamViewWrapper role="table">
- <ScrollableLogTextStreamViewWrapper>
```

```diff
! scrollable_log_text_stream_view.tsx#225

- aria-owns="streamEntry logTextStreamEntry"
```